website/docs/create-your-composite.mdx

```diff
@@ -24,38 +26,40 @@ Here, the flag `--table` will display the output in an organized table view and
 
 **Response:** Below see a small snippet of the the output table. Each model has the following properties:
 
--  `Name` - model name
--  `Unique ID` - unique identifier (stream ID) for the model
--  `Description` - description of the model’s functionality
-
-![Data Model Table](../static/img/data-model-table.png)
+- `Name` - model name
+- `Unique ID` - unique identifier (stream ID) for the model
+- `Description` - description of the model’s functionality
 
+![Data Model Table](/img/data-model-table.png)
 
 ## Creation
+
 In this section we will demonstrate how to create a composite by downloading models from the model catalog.
 
 ### Single model
+
 You can fetch any existing model from the catalog by referencing the model’s unique ID. For example, for your basic social media app, use the existing model `SimpleProfile`. To fetch the model, to your working directory, take note of the model stream ID in the table above and run the following command:
 
 ```bash
-composedb composite:from-model kjzl6hvfrbw6c7keo17n66rxyo21nqqaa9lh491jz16od43nokz7ksfcvzi6bwc --ceramic-url=http://localhost:7007 --output=my-first-composite-single.json
+composedb composite:from-model kjzl6hvfrbw6c5ajfmes842lu09vjxu5956e3xq0xk12gp2jcf9s90cagt2god9 --ceramic-url=http://localhost:7007 --output=my-first-composite-single.json
```

Comment:
Nice, should we update the streamids in the guide's `@loadModel` examples as well?
https://composedb.js.org/docs/preview/guides/data-modeling/relations#example-post-with-comments

Reply:
I don't know where these model IDs are defined, are they supposed to be working?
Seems they got added in the preview docs since they are not in https://composedb.js.org/docs/0.3.x/guides/creating-composites/directives#relations? cc @3avi

Reply:
Ok, would be good to make sure that these examples work. Quite likely that people will try them

Reply:
noted - will update so it's a working example